src/slack/publishMessages.js

```diff
@@ -25,11 +32,11 @@ module.exports = async ({ channel, repository, branches }) => {
 
   const blocks = composeBlocks({ repository, branches });
 
-  // eslint-disable-next-line no-plusplus
-  for (let i = 0; i < blocks.length; i++) {
-    // eslint-disable-next-line no-await-in-loop
-    await postMessage({ channel, blocks: [blocks[i]] });
-  }
+  blocks.forEach((block) => postMessage({ channel, blocks: [block] }));
```

Comment:
I think this forEach can replace the for loop and not have the eslint-disables - am I wrong? Does slack js engine not have forEach implemented?

Reply:
Yeah, I also prefer `forEach ` in most cases but if my memory serves me correctly, the asynchronous loop has been used to post Slack messages in order oldest branch first. When `forEach` is used instead, we will get the messages to show up on Slack like this 😞 .

![image](https://user-images.githubusercontent.com/5466341/95031726-7a777c00-0685-11eb-8669-9b78f0e416ec.png)

Reply:
oh that `forEach` in Slack isn't in order 😢 That's weird of slack, https://stackoverflow.com/a/13601073/962292 seems like the language spec & MDN both say `forEach` iterates in ascending index order. I'll revert and put a comment so nobody else is confused in the future.

Reply:
Interesting... I guess it could also have something to do with the `await` keyword used in the for loop.